tests/Unit/Schema/AST/ASTBuilderTest.php

```diff
@@ -58,4 +125,54 @@ public function testDoesNotAllowDuplicateFieldsOnTypeExtensions(): void
         $this->expectException(DefinitionException::class);
         $this->astBuilder->documentAST();
     }
+
+    public function testDoesNotAllowDuplicateFieldsOnInputExtensions(): void
+    {
+        $this->schema = '
+        input Inputs {
+            foo: String
+        }
+
+        extend input Inputs {
+            foo: Int
+        }
+        ';
+
+        $this->expectException(DefinitionException::class);
```

Comment:
What are the actual produced error messages? Are they somewhat understandable?

Reply:
I think so, it will show something like:
```
Nuwave\Lighthouse\Exceptions\DefinitionException: Duplicate definition TWO found when merging
```